### DISCUSSION TITLE
Organization Features?

### DISCUSSION THREAD

**Discussion by OfficialVannt** [xivdev/Penumbra#197] [Ideas] [answered]
Would be nice to create folders within penumbra to categorize what mods do what, right now my installed mods tab is just filled with a mixture of body, clothing and misc. making it harder to navigate without proper folder organization.

**Comment by SerenitySublime** [comment, marked as answer]
You can right click a mod, and put a folder before it to make a folder. Like if you have a mod called Sweater, and you wanna put it in a Clothes folder, you would type Clothes/Sweater, and it would make a folder for you called Clothes and put the Sweater mod in it.

**Reply by OfficialVannt** [reply]
I really wish it did what you said, but it doesn't. Right-clicking a mod will bring up the "sort order" button and naming the mod under something like "Clothes" does nothing. The mod is still called the original name and there is no new folder, but when I right-click that specific mod, it just reads as "Clothes"

**Reply by OfficialVannt** [reply]
Update: I'm retarded. Thank you for the reply.

**Reply by Ottermandias** [reply]
Click the ? button at the bottom of the mod selector for detailed info.
